lib/shared/log/toConsole.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var fancyLog = require('fancy-log');
+
+// The sorting of the levels is
+// significant.
+var levels = [
+  'debug', // Logs all levels.
+  'info',  // Logs info, warn and error.
+  'warn',  // Logs warn and error.
+  'error', // Logs error.
```

Comment:
@phated in this case, we should refactor `fancy-log` to handle errors with `console.error` right?